docker/daemon_unix.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/pkg/system"
 
-	_ "github.com/docker/docker/daemon/execdriver/lxc"
+	//"github.com/docker/docker/daemon/execdriver/lxc"
```

Comment:
I dont really get the point of that line, linux build seems to build ok without that.